### DISCUSSION TITLE
Building from source

### DISCUSSION THREAD

**Discussion by troyBORG** [vrcx-team/VRCX#149] [Q&A] [answered]
I was just curious about trying to build from source, do I just do the [_build-node.js.cmd_](https://i.imgur.com/oFILGUz.png), or what do I all need to do to get an exe file?

**Comment by Natsumi-sama** [comment, marked as answer]
https://github.com/pypy-vrc/VRCX#how-to-build-vrcx once built with Visual studio the exe will be located in ``.\VRCX\bin\x64\Release\``
